spark-bigquery-dsv2/spark-3.4-bigquery/src/test/java/com/google/cloud/spark/bigquery/integration/Spark34IndirectWriteIntegrationTest.java

```diff
@@ -31,5 +50,47 @@ public void setParquetLoadBehaviour() {
   }
 
   // tests from superclass
+  @Test
+  public void testTimestampNTZWriteToBigQueryAvroFormat() throws InterruptedException {
```

Comment:
Please create a provate method that accepts the intermediate format instead of duplicating the code

Reply:
Done